worker/src/db_handler.rs

```diff
@@ -42,4 +80,58 @@ impl DBHandler {
             }
         });
     }
+
+    pub fn send_data_to_enclave(eid: sgx_enclave_id_t) -> Result<(), PolkadexDBError> {
+        let balances = load_balances_mirror()?
+            .lock()
+            .map_err(|_| PolkadexDBError::UnableToLockMutex)?;
+        let nonce = load_nonce_mirror()?
+            .lock()
+            .map_err(|_| PolkadexDBError::UnableToLockMutex)?;
+        let orderbook = load_orderbook_mirror()?
+            .lock()
+            .map_err(|_| PolkadexDBError::UnableToLockMutex)?;
+
+        let balances_data = balances.prepare_for_sending()?;
+        let nonce_data = nonce.prepare_for_sending()?;
+        let orderbook_data = orderbook.prepare_for_sending()?;
+
+        let (mut balances_chunks, mut nonce_chunks, mut orderbook_chunks) = (
+            balances_data.chunks(1000),
```

Comment:
It would make sense here to define a Constant instead of hardcoded values somewhere in the code.

Reply:
Done.